### ISSUE TITLE
extra documentation for weighting module (Trac #1466)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1458]
It would be useful to add another section to 

[http://software.icecube.wisc.edu/simulation_trunk/projects/weighting/tutorial.html]

that explains that you need to add:

tray.AddModule(get_weighted_primary,'get_weighted_primary')

when creating the hdf5 files. Otherwise events with multiple primaries will have n_weights < n_primaries and this causes inf's and nan's in the weights if you only follow the tutorial blindly. Easy to identify the problem, but not the solution (as a newcomer).

Migrated from https://code.icecube.wisc.edu/ticket/1466
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:45", 
    "description": "It would be useful to add another section to \n\n[http://software.icecube.wisc.edu/simulation_trunk/projects/weighting/tutorial.html]\n\nthat explains that you need to add:\n\ntray.AddModule(get_weighted_primary,'get_weighted_primary')\n\nwhen creating the hdf5 files. Otherwise events with multiple primaries will have n_weights < n_primaries and this causes inf's and nan's in the weights if you only follow the tutorial blindly. Easy to identify the problem, but not the solution (as a newcomer).\n\n", 
    "reporter": "summer.blot", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067105393059", 
    "component": "cmake", 
    "summary": "extra documentation for weighting module", 
    "priority": "normal", 
    "keywords": "weighting", 
    "time": "2015-12-04T15:58:00", 
    "milestone": "", 
    "owner": "jvansanten", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20151209T15:33:21: olivas changed milestone from "" to "IceSim 5.0"

**Comment by icecube-trac** [comment]
Trac update at 20151216T13:22:45: 
* jvansanten changed resolution from "" to "fixed"
* jvansanten changed status from "new" to "closed"
* jvansanten commented:

> In changeset 140516:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="140516"
> Explicitly mention that MCPrimary should be a single-row table. Closes #1458
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:45: 
* olivas changed milestone from "IceSim 5.0" to ""
* olivas commented:

> Milestone IceSim 5.0 deleted
